src/adapter.c

```diff
@@ -437,13 +437,13 @@ int close_adapter(int na)
 	}
 
 	LOG("closing adapter %d  -> fe:%d dvr:%d, sock:%d, fe_sock:%d", na, ad->fe, ad->dvr, ad->sock, ad->fe_sock);
-	ad->enabled = 0;
```

Comment:
Hi @catalinii ,

You can't do it here as then the find_adapter() function used by the mark_pids_deleted() will return FALSE.